backend/models/events.py

```diff
@@ -10,8 +10,9 @@
 
 class OpsEventType(Enum):
```

Comment:
Curious - If we use a `IntEnum`, would we need to specify the id for each item still? I'm thinking to the future, as I know my Agreement feature branch, I already created a `CREATE_NEW_AGREEMENT = 4`; as we continue to add to this, I can see lots of conflict 🤔

Reply:
Discussed, and resovled